### ISSUE TITLE
xUnit1033 incorrect detection

### ISSUE THREAD

**Issue by bdovaz** [xunit/xunit#2567]
@bradwilson

First, the page does not exist:

https://xunit.net/xunit.analyzers/rules/xUnit1033

https://xunit.net/xunit.analyzers/rules/

If I have this code, it does not detect it correctly:

```
Test class 'MyTest' does not contain constructor argument of type 'MyFixture'. Add a constructor argument to consume the fixture data.
```

```csharp
public MyTest(MyFixture fixture, ITestOutputHelper output)
{
}
```

It seems that it only expects a constructor with one parameter... But ITestOutputHelper is a parameter that can be passed along with the fixture:

https://xunit.net/docs/capturing-output

The exact problem it's here:

https://github.com/xunit/xunit.analyzers/blob/main/src/xunit.analyzers/TestClassShouldHaveTFixtureArgument.cs#L47

**Comment by bdovaz** [comment]
Created a PR https://github.com/xunit/xunit.analyzers/pull/150

**Comment by WernerMairl** [comment]
same here - happy to see the PR merged!

**Comment by bradwilson** [comment]
PR was merged. Thanks!

**Comment by bradwilson** [comment]
This will be part of the `1.1.0` release of the analyzers. ([Pre-release builds](https://myget.org/feed/xunit/package/nuget/xunit.analyzers) can be found on our [MyGet Gallery](https://myget.org/gallery/xunit)); this was part of `1.1.0-pre.2`.

**Comment by bdovaz** [comment]
@bradwilson do you have an ETA for 1.1.0? Thanks.

**Comment by bradwilson** [comment]
No ETA. You can use the 1.1.0 pre-release builds from MyGet until it does.

**Comment by bdovaz** [comment]
@bradwilson The problem also lies in the fact that xunit depends on xunit.analyzers >= 1.0.0.

That's why I can neither downgrade nor remove the analyzers package...

This dependency exists for convenience only, isn't it? I mean, in reality there is no such dependency.

I have my doubts that this is correct, the analyzers should be something opt-in to give the possibility to add or remove it at least.

**Comment by bradwilson** [comment]
The 1.1.0 pre-release version satisfies `>= 1.1.0`.

`xunit` is a meta-package. If you don't want all three of the dependencies that it brings, then remove `xunit` and add the ones you want instead (we give the same advice for anybody who doesn't want to use our assertion library).

**Comment by bradwilson** [comment]
@bdovaz I don't currently have any schedule for the next release. Let me see what's already post `1.0.0` and check to see what other new stuff could be added/fixed for a `1.1.0` release.

**Comment by bradwilson** [comment]
@bdovaz @bdominguez I just pushed up 1.1.0: https://www.nuget.org/packages/xunit.analyzers/1.1.0
